zcash_primitives/Cargo.toml

```diff
@@ -42,11 +42,15 @@ subtle = "2.2.3"
 bls12_381 = "0.8"
 ff = "0.13"
 group = { version = "0.13", features = ["wnaf-memuse"] }
-incrementalmerkletree = "0.3"
 jubjub = "0.10"
 nonempty = "0.7"
 orchard = { version = "0.4", default-features = false }
 
+# - Note Commitment Trees
+bridgetree = "0.2"
```

Comment:
Is this dependency used any more? I thought the `bridgetree` APIs were all being moved to `zcashd`?

Reply:
Some serialization remains here until it's moved out in a later commit. This updates the build & tests in preparation for it being moved out.

Reply:
Will that later commit be added to this PR, or will it be in a subsequent PR? If the latter, we either need to ensure that PR is merged before the next release, or fix the changelog to not say this dependency is removed.

Reply:
This can now be removed.